<launch>

  <arg name="display_output" default="true"/>
  <arg name="camera_topic" default="/usb_cam/image_raw"/>
  <arg name="chip_type" default="rk3588"/>
  <arg name="model_name" default="yolov8m.rknn"/>

  <node name="rknn_yolov8_ros" pkg="rknn_yolov8_ros" type="rknn_yolov8_node" output="screen">
    <param name="model_name" value="$(arg model_name)"/>
    <param name="display_output" value="$(arg display_output)"/>
    <param name="box_conf_threshold" value="0.35"/>
    <param name="nms_threshold" value="0.45"/>
    <param name="chip_type" value="$(arg chip_type)"/>
    <remap from="camera_topic" to="$(arg camera_topic)"/>
  </node>
</launch>
